dto/api_input.py

```diff
@@ -72,34 +79,42 @@ class TeamsStatisticsOrPlayersDTO(BaseDTO):
     name = fields.Str(required=False, missing=None)
 
 
-class SearchDTO(BaseDTO):
-    leagues__sport_id = fields.Int(required=False, missing=None)
-    countries__country_id = fields.Int(required=False, missing=None)
-    letter = fields.Str(required=False, missing="")
+class PaginationDTO(BaseDTO):
     page = fields.Int(required=False, missing=0)
     per_page = fields.Int(required=False, missing=0)
 
+class FilterDTO(BaseDTO):
+    filter_name = fields.Str(required=True)
+    filter_value = fields.Raw(required=True)
+    order = fields.Str(required=False, missing=None)
```

Comment:
I saw you using order_type and order_field or something like that in your FilterManager

do the same here please